<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommitBase</class>
 <widget class="QWidget" name="CommitBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commit changes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/resources/svn-commit.svg</normaloff>:/icons/resources/svn-commit.svg</iconset>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="treeWidgetFiles">
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
        </property>
        <property name="toolTip">
         <string>Check files to commit</string>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <column>
         <property name="text">
          <string>File</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Index status</string>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabelMsg">
           <property name="text">
            <string>Commit message (first line is the sub&amp;ject):</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>textEditMsg</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditMsg">
           <property name="toolTip">
            <string>Lines starting with '#' will be ignored
Press &lt;Ctrl+Enter&gt; to commit changes</string>
           </property>
           <property name="lineWrapMode">
            <enum>QTextEdit::LineWrapMode::NoWrap</enum>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabelLineCol">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
         <property name="text">
          <string>Line: %1 Col: %2</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Settin&amp;gs</string>
         </property>
         <property name="shortcut">
          <string>Alt+G</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Canc&amp;el</string>
         </property>
         <property name="shortcut">
          <string>Alt+E</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUpdateCache">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Update index for selected files</string>
         </property>
         <property name="text">
          <string>Update &amp;Index</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOk">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Commit selected files</string>
         </property>
         <property name="text">
          <string>&amp;Commit</string>
         </property>
         <property name="shortcut">
          <string>Alt+C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonUpdateCache</tabstop>
  <tabstop>pushButtonOk</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CommitBase</receiver>
   <slot>pushButtonCancel_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonUpdateCache</sender>
   <signal>clicked()</signal>
   <receiver>CommitBase</receiver>
   <slot>pushButtonUpdateCache_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSettings</sender>
   <signal>clicked()</signal>
   <receiver>CommitBase</receiver>
   <slot>pushButtonSettings_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButtonCancel_clicked()</slot>
  <slot>pushButtonUpdateCache_clicked()</slot>
  <slot>pushButtonSettings_clicked()</slot>
 </slots>
</ui>
